### PR TITLE
[a11y] Fix "read more" occurrence

### PR DESCRIPTION
**Pull request by alaudazzi**
This PR replaces "read more" with accessible content.

Rel: #5306

The generated content in `../reference/api-docs.asciidoc` is not in scope.

### CODE REVIEW

**Comment by thbkrkr** [comment]
Should we also replace the occurrences of "read the"?
```sh
> grep ' [Rr]ead the' -r docs
docs/orchestrating-elastic-stack-applications/security/saml-authentication.asciidoc:TIP: Make sure you read the complete ...
docs/quickstart.asciidoc:IMPORTANT: Read the <<{p}-upgrading-eck,upgrade notes>>  ...
docs/operating-eck/installing-eck.asciidoc: Read the <<{p}-ga-upgrade,upgrade documentation>> ...
```

**Comment by alaudazzi** [comment]
@thbkrkr 
The last commit fixes more "read" occurrences.

File: docs/release-notes/1.7.0.asciidoc [hunk 1]
```diff
@@ -35,7 +35,7 @@
 [float]
 === Bug fixes
 
-* Beats: allow unprivileged Pods to read configuration file {pull}4572[#4572] (issue: {issue}4562[#4562])
+* Beats: allow unprivileged Pods to access the configuration file {pull}4572[#4572] (issue: {issue}4562[#4562])
```
Comment (thbkrkr) [inline docs/release-notes/1.7.0.asciidoc:38]:
```suggestion
* Beats: allow unprivileged Pods to read configuration file {pull}4572[#4572] (issue: {issue}4562[#4562])
```

File: docs/design/0005-configurable-operator.md [hunk 2]
```diff
@@ -135,7 +135,7 @@ Pros:
 Cons:
 
 * Need for a "hybrid" version to deploy everything in a single restricted namespace (eg. "default"), which is a bit confusing
-* The global-operator needs elevated permissions on the cluster (eg. read access to all secrets)
+* The global-operator needs elevated permissions on the cluster (eg. get access to all secrets)
```
Comment (thbkrkr) [inline docs/design/0005-configurable-operator.md:138]:
```suggestion
* The global-operator needs elevated permissions on the cluster (eg. read access to all secrets)
```

**Review by thbkrkr — COMMENTED** [review]
I think we shouldn't replace the occurrences when the `read` doesn't apply to the reader.

**Review by thbkrkr — APPROVED** [review]
LGTM

**Pull request merged**